lib/src/_shared/DateTextField.jsx

```diff
@@ -42,6 +44,8 @@ export default class DateTextField extends PureComponent {
     mask: undefined,
     keyboardIcon: 'event',
     invalidDateMessage: 'Invalid Date Format',
+    clearable: true,
+    onClear: undefined,
```

Comment:
I think we can make `onClear` required (like `onChange`), since it will be always passed from `ModalWrapper`

Reply:
Same as for clearable. I can change it if requested.